java/clients/src/main/java/sleeper/clients/QueryWebSocketClient.java

```diff
@@ -230,15 +245,15 @@ public void onMessage(String json) {
                     out.println("Query results:");
                     records.values().stream()
                             .flatMap(List::stream)
-                            .forEach(record -> out.println(record));
+                            .forEach(record -> out.println(record.toString()));
```

Comment:
This is changing the behaviour of the command line client significantly. Can we still output the JSON please?

Reply:
I dont think it does? The WebSocketResultsOutput class was serialising records by calling `toString()` on them before. Now we are deserialising the Record objects, this is needed to keep the old functionality.